### PR TITLE
Add xpore

### PR DESCRIPTION
**Pull request by drosofff**
xpore is a python package for Nanopore data analysis of differential RNA modifications

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by drosofff** [comment]
@BiocondaBot please add label

File: recipes/xpore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2478c434f40aa24e901fc177a726f2f90fbf09b87ac530cdbc23908729c907be
+
+build:
+  number: 0
+  noarch: generic
```
Comment (bgruening) [inline recipes/xpore/meta.yaml:14]: noarch: python

File: recipes/xpore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  entry_points:
+    - xpore-dataprep=xpore.scripts.dataprep:main
+    - xpore-diffmod=xpore.scripts.diffmod:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - h5py >=2.10.0
```
Comment (bgruening) [inline recipes/xpore/meta.yaml:22]: remove everything from the host section that is not, python or pip

File: recipes/xpore/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - xpore-dataprep --help
+    - xpore-diffmod --help
+
+about:
+  home: "https://github.com/GoekeLab/xpore"
+  license: MIT
+  license_family: MIT
+  license_file:
+  summary: "xpore is a python package for Nanopore data analysis of differential RNA modifications."
+  doc_url:
```
Comment (bgruening) [inline recipes/xpore/meta.yaml:55]: can you add this or remove it?

File: recipes/xpore/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - xpore.utils
+  commands:
+    - xpore-dataprep --help
+    - xpore-diffmod --help
+
+about:
+  home: "https://github.com/GoekeLab/xpore"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (bgruening) [inline recipes/xpore/meta.yaml:53]: can you please add a license file?

**Review by bgruening — COMMENTED** [review]
Thanks @drosofff

File: recipes/xpore/meta.yaml (hunk 1 above)
Comment (drosofff) [reply recipes/xpore/meta.yaml]: done

File: recipes/xpore/meta.yaml (hunk 2 above)
Comment (drosofff) [reply recipes/xpore/meta.yaml]: ok

File: recipes/xpore/meta.yaml (hunk 3 above)
Comment (drosofff) [reply recipes/xpore/meta.yaml]: yes

File: recipes/xpore/meta.yaml (hunk 4 above)
Comment (drosofff) [reply recipes/xpore/meta.yaml]: stated "LICENCE" as in the dev github repo. Am I required to add here the *file* ?

**Comment by drosofff** [comment]
Thanks @bgruening I also add some extra identifiers

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Pull request merged**